preproc/ft_preproc_dftfilter.m

```diff
@@ -86,28 +92,58 @@
 %
 % $Id$
 
-% defaults
-Flreplace = ft_getopt(varargin,'dftreplace','zero');
-Flwidth = ft_getopt(varargin, 'dftbandwidth', [1 2 3]);
-Neighwidth = ft_getopt(varargin, 'dftneighbourwidth', [2 2 2]);
+% Modified by Richard J. Cui on: Sun 03/29/2020  4:40:10.909 PM;
```

Comment:
We don't take comments in the code like this. Your contribution is visible in the git commit message.